packages/fx-core/tests/plugins/resource/sql/helper.ts

```diff
@@ -28,6 +28,10 @@ export class TestHelper {
           console.debug(message);
           return true;
         },
+        async warning(message: string): Promise<boolean> {
```

Comment:
when is this method called seems not in this pr?

Reply:
it is a mock for logProvider for test.
It is used in ctx.logProvider?.warning()